drake/systems/LCMSystem.cpp

```diff
@@ -0,0 +1,55 @@
+
+#include <iostream>
+#include "LCMSystem.h"
+
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
agreed.  this was old code (copied in).  but i've changed it now to just check _WIN32.